src/Foundation/Exception/Handler.php

```diff
@@ -2,13 +2,15 @@
 
 use Log;
 use Event;
+use ReflectionClass;
 use Response;
 use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
 use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
 use October\Rain\Exception\AjaxException;
 use ReflectionFunction;
 use Exception;
 use Closure;
+use Throwable;
```

Comment:
@bennothommo can you octoberify this use block?

Reply:
@LukeTowers done :)